src/dbt/kipptaf/tests/test_new_grad_path_combo.sql

```diff
@@ -0,0 +1,12 @@
+{{-
+    config(
+        severity="warn",
+        store_failures=true,
+        store_failures_as="view",
+        enabled=true,
+    )
+-}}
+
+select *,
+from {{ ref("model_name") }}

```

Comment:
doesn't reference a real model, and this will break the project since it doesn't exist